packages/next/test/integration/test-limit-exceeded-404-static-files/next.config.js

```diff
@@ -23,7 +23,7 @@ assert(
 // generate 100MB text file which will be traced in `/api/hello`
```

Comment:
Should this comment say 200MB now?

Reply:
Yea will make this comment generic though since arbitrary size